modules/openapi-generator/src/main/resources/go-experimental/README.mustache

```diff
@@ -54,19 +54,14 @@ Class | Method | HTTP request | Description
 {{/authMethods}}{{#authMethods}}{{#last}} Authentication schemes defined for the API:{{/last}}{{/authMethods}}
 {{#authMethods}}
 
-## {{{name}}}
+### {{{name}}}
 
-{{#isApiKey}}- **Type**: API key
+{{#isApiKey}}
+- **Type**: API key
+- **API key parameter name**: {{{keyParamName}}}
+- **Location**: {{#isKeyInQuery}}URL query string{{/isKeyInQuery}}{{#isKeyInHeader}}HTTP header{{/isKeyInHeader}}
 
-Example
-
-```golang
-auth := context.WithValue(context.Background(), sw.ContextAPIKey, sw.APIKey{
-    Key: "APIKEY",
-    Prefix: "Bearer", // Omit if not necessary.
-})
-r, err := client.Service.Operation(auth, args)
-```
+Note, each API key must be added to a map of `map[string]APIKey` where the key is: {{keyParamName}} and passed in as the auth context for each request.
```

Comment:
@nmuesch minor feedback. Is there a way we can keep the sample code?

It would be great to show the code to use the API key (single or multiple)

Reply:
Hey, thanks for the feedback. There was a brief discussion of this here https://github.com/OpenAPITools/openapi-generator/pull/3450#pullrequestreview-277692301 interested in your thoughts. The main reason I changed it was because it made the `example` code incorrect for any apps that have multiple keys. I couldn't figure out a way to show the full multi-key example using the template variables.

Reply:
We can do it in the actual code sample instead and we can do it in another PR instead of this one as I don't want this PR to be opened for too long